packages/jaeger-ui/src/components/SearchTracePage/SearchResults/DiffSelection.css

```diff
@@ -32,3 +32,7 @@ limitations under the License.
   border: 1px solid #ccc;
   border-bottom: none;
 }
+
+.DiffSelection--message button {
```

Comment:
It would be better to add a class name to these buttons and then you could select them with:
`.DiffSelection--cohortBtn ~ .DiffSelection--cohortBtn {`

the `~` should mean that any `cohortBtn` following a `cohortBtn` would have the `margin-left` styling. if `~` doesn't work, you can replace the selector with `.DiffSelection--cohortBtn:not(:first-of-type)` should also work but has slightly worse browser compatibility (though we usually aren't incredibly concerned with supporting super old browsers).